app/app.go

```diff
@@ -591,6 +606,18 @@ func NewWasmApp(
 		keys[jwktypes.StoreKey],
 		app.GetSubspace(jwktypes.ModuleName))
 
+	govModAddress := authtypes.NewModuleAddress(govtypes.ModuleName).String()
```

Comment:
This is inlined in every single other Keeper. If we are going to use this model, it should be consistent.